lib/rules/space-after-keywords.js

```diff
@@ -23,6 +23,10 @@ module.exports = function(context) {
      * @returns {void}
      */
     function checkTokens(node, left, right) {
+        if (right.type !== "Punctuator") {
```

Comment:
Let's go with this and see what happens. We can always fix an extra message later if necessary.